src/test/resources/io/cucumber/features/FileDownload.feature

```diff
@@ -0,0 +1,12 @@
+@api
+Feature: File download
+
+  Scenario: Download file
+    Given I have a File
```

Comment:
Give I have access to a URL for a file to download

Reply:
Not the best wording-- but trying to demonstrate that if you have a file, you don't need to download it.

Reply:
Resolved in next check in